modules/cactus-web/src/Select/Select.tsx

```diff
@@ -163,7 +163,7 @@ function asOption(opt: string | OptionType): OptionType {
 }
 
 function getOptionId(option: OptionType) {
-  return `${option.label}-${option.value}`.replace(/\s/g, '-')
+  return `${option.label}-${option.value}`.replace(/\s/g, '-').replace(/:|\.|\//g, '')
```

Comment:
We could write this regex a little more efficiently if we combine them 
```suggestion
  return `${option.label}-${option.value}`.replace(/[\s:.\/]/g, '')
```

Reply:
So you're okay with just removing the spaces altogether instead of replacing them with dashes then?